src/stats/inc/GaussianLikelihoodFullCovarianceRandomCoefficient.h

```diff
@@ -36,7 +36,9 @@ class GslMatrix;
  * \file GaussianLikelihoodFullCovarianceRandomCoefficient.h
  *
  * \class GaussianLikelihoodFullCovarianceRandomCoefficient
- * \brief A class that represents a Gaussian likelihood with full covariance and random coefficient
+ * \brief A class that represents a Gaussian likelihood with full covariance
+ * and random coefficient, treated as a hyperparameter to be inferred during
```

Comment:
Is every coefficient in the matrix being treated as an independent parameter? If so, I might spell that out explicitly.